ads/google/a4a/utils.js

```diff
@@ -156,6 +165,12 @@ export function googleAdUrl(
         {name: 'brdim', value: additionalDimensions(win, viewportSize)},
         {name: 'isw', value: viewportSize.width},
         {name: 'ish', value: viewportSize.height},
+        // TODO(levitzky) Look into expandable support.
+        {name: 'ea', value: '0'},
+        {name: 'pfx', value:
+          containerTypeSet[ValidAdContainerTypes['AMP-FX-FLYING-CARPET']]
+              || containerTypeSet[ValidAdContainerTypes['AMP-STICKY-AD']]
+              || false},
```

Comment:
Without the '... || false', the expression would evaluate to undefined if there's no match, rather than false, which is what Bow expects.